src/Tag/Base/NotEncodeStringableInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag\Base;
+
+interface NotEncodeStringableInterface
```

Comment:
Currently it's implemented by `Tag` only. Is it meant to be implemented by something else? If so, what?

Reply:
Naming considerations:

1. NoEncodeStringableInterface
2. If it's a universal interface, namespace should be `Yiisoft\Html`.

Reply:
Interface renamed and moved to root.

This interface used in `Tag` and widgets `CheckboxList` and `RadioList`.